external/anomaly/anomaly_classification/task.py

```diff
@@ -154,9 +155,15 @@ def save_model(self, output_model: ModelEntity):
         output_model.set_data("weights.pth", buffer.getvalue())
         output_model.set_data("label_schema.json", label_schema_to_bytes(self.task_environment.label_schema))
         # store computed threshold
-        output_model.set_data("threshold", bytes(struct.pack("f", self.model.threshold.item())))
-
-        f1_score = self.model.image_metrics.OptimalF1.compute().item()
+        output_model.set_data("threshold", bytes(struct.pack("f", self.model.image_threshold.value.item())))
+        output_model.set_data("pixel_threshold", bytes(struct.pack("f", self.model.pixel_threshold.value.item())))
```

Comment:
Similar to what you've done in Lines 160-164, would it be possible to follow the same convention here to convert Lines 158-159 to bytes. L160-164 looks simpler to read.

Reply:
Something like

```
output_model.set_data("threshold", self.model.image_threshold.value.cpu().numpy().tobytes())
output_model.set_data("pixel_threshold", self.model.pixel_threshold.value.cpu().numpy().tobytes())
```